include/seqan3/alphabet/detail/debug_stream_alphabet.hpp

```diff
@@ -30,7 +29,7 @@ namespace seqan3
 template <typename char_t, alphabet alphabet_t>
 inline debug_stream_type<char_t> & operator<<(debug_stream_type<char_t> & s, alphabet_t && l)
 //!\cond
-    requires (!output_stream_over<std::basic_ostream<char_t>, alphabet_t>)
+    requires (!std::integral<std::remove_cvref_t<alphabet_t>>)
```

Comment:
is this really equivalent?

Reply:
It is not equivalent, but it does what it is supposed to at that point ^^